package/src/utils.cpp

```diff
@@ -95,4 +94,13 @@ const std::pair<int, std::vector<std::string>> cmd_pipe_output(
   return ret;
 }
 
-}  // namespace prmon
+const void prmon::fill_units(nlohmann::json& unit_json,
+                             const parameter_list& params) {
+  for (const auto& param : params) {
+    if (!((param.get_max_unit()).empty()))
```

Comment:
I think we have a redundant set of parentheses here, not that it really matters :)